java/semantickernel-core/src/test/java/com/microsoft/semantickernel/memory/VolatileMemoryStoreTests.java

```diff
@@ -93,17 +93,13 @@ void itCanCreateAndGetCollectionAsync() {
     }
 
     @Test
-    void itCannotCreateDuplicateCollectionAsync() {
+    void  itHandlesExceptionsWhenCreatingCollectionAsync() {
         // Arrange
-        String collection = "test_collection" + this._collectionNum;
-        this._collectionNum++;
-
-        // Act
-        this._db.createCollectionAsync(collection).block();
+        String collection = null;
 
         // Assert
         assertThrows(
-                MemoryException.class,
+                NullPointerException.class,
```

Comment:
seems a bit odd to expect an NPE

Reply:
This change was made to match with recent changes in .NET